flybywiresim/a32nx
# fix(efb): UI fixes, dev server

## Summary of Changes


* Use a common navbar component
  * Fixes some navbars not having hover styling
  * Removes unnecessary code execution 
* Load font correctly
* Fix dashboard heading size
* Fix letter spacing
* Add rollup-based development server with live-reloading to run EFB in browser


Discord username (if different from GitHub): someperson#4953

## Testing instructions


* Make sure UI is functional
* Make sure navbar links are FBW blue on hover


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
